html/index.html

```diff
@@ -205,111 +205,144 @@ <h2 class="my-0 fw-semibold">Nautobot Core</h2>
-                                    class="link-dark">Nautobot Data Validation Engine</a><span
+                                    href="https://docs.nautobot.com/projects/plugin-nornir/en/stable/"
+                                    class="link-dark">Nautobot Plugin Nornir</a><span
                                     class="badge text-bg-dark rounded-pill ms-1">App</span>
                             </li>
                             <li class="list-group-item"><a
-                                    href="https://docs.nautobot.com/projects/netbox-importer/en/latest/"
-                                    class="link-dark">Nautobot NetBox Importer</a><span
+                                    href="https://docs.nautobot.com/projects/circuit-maintenance/en/stable/"
+                                    class="link-dark">Nautobot Circuit Maintenance</a><span
                                     class="badge text-bg-dark rounded-pill ms-1">App</span>
                             </li>
                             <li class="list-group-item"><a
-                                    href="https://docs.nautobot.com/projects/welcome-wizard/en/latest/"
+                                    href="https://docs.nautobot.com/projects/welcome-wizard/en/stable/"
                                     class="link-dark">Nautobot Welcome Wizard</a><span
                                     class="badge text-bg-dark rounded-pill ms-1">App</span>
                             </li>
                             <li class="list-group-item"><a
-                                    href="https://docs.nautobot.com/projects/circuit-maintenance/en/latest/"
-                                    class="link-dark">Nautobot Circuit Maintenance</a><span
+                                    href="https://docs.nautobot.com/projects/capacity-metrics/en/stable/"
+                                    class="link-dark">Nautobot Capacity Metrics</a><span
                                     class="badge text-bg-dark rounded-pill ms-1">App</span>
                             </li>
+                            <li class="list-group-item"><a
+                                    href="https://docs.nautobot.com/projects/netbox-importer/en/stable/"
+                                    class="link-dark">Nautobot NetBox Importer</a><span
+                                    class="badge text-bg-dark rounded-pill ms-1">App</span>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="col">
+                    <div class="card mb-4 rounded-3 shadow-sm h-100">
+                        <div class="card-header py-3">
+                            <h2 class="my-0 fw-semibold">Commercial Software</h2>
+                        </div>
+                        <div class="card-body">
+                            <p class="card-text">This section links to documentation for commercial Nautobot Products.
```

Comment:
One nitpick to fulfill my quota.
```suggestion
                            <p class="card-text">This section links to documentation for commercial Nautobot products.
```